packages/solid/src/generators/application/schema.d.ts

```diff
@@ -0,0 +1,22 @@
+import { Linter } from '@nrwl/linter';
+
+export interface SolidApplicationSchema {
+  name: string;
+  tags?: string;
+
+  linter: Linter;
+  unitTestRunner: 'jest' | 'none';
+  e2eTestRunner: 'cypress' | 'none';
+  directory?: string;
+  host?: string;
+  port?: number;
+  bundler?: 'vite';
```

Comment:
Vite is the only option, because of that, I'd remove the bundler property. (Or do you plan more? What's supported by solidjs?

Reply:
Agreed. I can see however that you can also configure solid to use `esbuild`, so I left it open for the future. Will remove it for now however.